vscode-dotnet-runtime-extension/package.json

```diff
@@ -51,6 +51,11 @@
 				"title": "Install the .NET SDK System-Wide.",
 				"category": ".NET Install Tool",
 				"enablement": "!dotnetAcquisitionExtension.isGlobalSDKUnsupported"
+			},
+            {
+				"command": "dotnet.uninstallPublic",
+				"title": "Uninstall .NET.",

```

Comment:
Commands in command palette don't end with period. Something needs to be fixed in all of above.
Also titles are not getting localized?

Reply:
Nothing in this extension is localized atm. This is planned for the future 😱